tasks/install.yml

```diff
@@ -108,9 +110,11 @@
   notify:
     - restart consul
   tags: installation
+  ignore_errors: "{{ ansible_check_mode }}"
 
 - name: Daemon reload systemd in case the binaries upgraded
-  systemd: daemon_reload=yes
```

Comment:
Thanks for the boy-scouting.

Reply:
@bbaassssiiee The value of the PR isn't here. What do you think about other changes? :wink:

Reply:
This is a valuable PR, please continue.